lib/diagnostic.js

```diff
@@ -27,7 +27,7 @@ let access = null; // replace null with your response
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-let scopes = [/* your response here */];
+let scopes = ['deathStar'];
```

Comment:
Per my notes above, `laserQuantity` is accessible from both the `deathStar` and `fire` execution contexts.